app/assets/stylesheets/landing_page/sections/listings.scss

```diff
@@ -0,0 +1,256 @@
+@import "../mixins/all";
+
+$listings__desktop-gutter: 24px;
+$listings__tablet-gutter: 16px;
+
+.listings__section {
+  color: $typography__main-color;
+
+  width: 100%;
+  @include prefix-val(display, flex);
+  @include prefix-prop(align-items, center);
+
+  padding: $dimension-section-margin 0px;
+}
+
+.listings__content {
+  margin: 0 auto;
+  text-align: center;
+
+  // mobile
+  width: 100%;
+
+  @media #{$tablet} {
+    width: 100%;
+  }
+
+  @media #{$desktop} {
+    max-width: 1114px + $listings__desktop-gutter;
+  }
+}
+
+.listings__title {
+  @include typography__section-title;
+}
+
+.listings__paragraph {
+  @include typography__section-paragraph;
+}
+
+.listings__listings {
+  @include prefix-val(display, flex);
+  @include prefix-prop(flex-direction, column);
+
+  // mobile
+  padding: 0px $dimension-mobile-margin;
+  margin-bottom: 36px;
+
+  @media #{$tablet} {
+    @include prefix-prop(flex-direction, row);
+    margin: 0;
+    padding: 0px (48px - $listings__tablet-gutter / 2) 48px (48px - $listings__tablet-gutter / 2);
+  }
+
+  @media #{$desktop} {
+    @include prefix-prop(flex-direction, row);
+    margin: 0;
+    padding: 0px (60px - $listings__desktop-gutter / 2) 60px (60px - $listings__desktop-gutter / 2);
+  }
+}
+
+.listings__listing {
+  margin: 0 auto;
+  width: 100%;
+  max-width: 327px;
+
+  margin-bottom: 24px;
+
+  @media #{$tablet} {
+    margin: 0;
+    padding: $listings__tablet-gutter / 2;
+    width: (100% / 3);
+  }
+
+  @media #{$desktop} {
+    margin: 0;
+    padding: $listings__desktop-gutter / 2;
+    width: (100% / 3);
+  }
+}
+
+.listings__listing-content {
+  background-color: #FFF;
+  width: 100%;
+  height: 100%;
+
+  @include prefix-val(display, flex);
+  @include prefix-prop(justify-content, space-between);
+  @include prefix-prop(flex-direction, column);
+
+  border-radius: $dimension-card-border-radius;
+  overflow: hidden;
+
+  box-shadow: 0px 2px 3px 0px rgba(0, 0, 0, 0.10);
+
+  &:hover {
+    transform: scale(1.011);
+    box-shadow: 0 0 50px 0 rgba(0,0,0,0.1);
+    @include prefix-prop(transition, all 50ms ease-in-out);
+  }
+}
+
+.listings__listing-image-container {
+  // width + padding-bottom makes a responsive square
+  width: 100%;
+  padding-bottom: 100%;
```

Comment:
Yep, and also possible to keep specific aspect rations, e.g. `padding-bottom: (100% / 16 * 9);` for 16:9 ratio